spec/abci/abci.md

```diff
@@ -237,6 +237,7 @@ via light client.
     - `Version (string)`: The Tendermint software semantic version
     - `BlockVersion (uint64)`: The Tendermint Block Protocol version
     - `P2PVersion (uint64)`: The Tendermint P2P Protocol version
+    - `AbciVersion (string)`: The Tendermint ABCI semantic version
```

Comment:
We should specify what "semantic version" actually means, I don't see any description of it anywhere in the spec.

Reply:
Also, shouldn't it be `ABCIVersion`, not `AbciVersion`?

Reply:
I left it as it would be generated in go without gogoproto. I changed it to `ABCI`.